packages/app/lib/src/features/home/data/post_repository.dart

```diff
@@ -46,6 +46,7 @@ final class _AppwritePostRepository implements PostRepository {
     required this.databaseId,
     required this.collectionId,
     required this.author,
+    // required this.feed,
```

Comment:
Why did you remove this, was it just becuase it was unused? (fair enough, just curious)

Reply:
You'd wanted to do conditional queries with it, but I'm always happy to delete dead code. I just thought you'd asked for it.

Reply:
Sorry, not that I can remember. I can't think of any feature that would need the data of the post the user just created.

Reply:
This comment was about passing the feed to the repo, not returning a post from createNewPost.